package.json

```diff
@@ -67,6 +67,7 @@
 		"lodash": "^4.17.15",
 		"log": "^1.4.0",
 		"md5": "^2.3.0",
+		"minimist": "^1.2.8",
```

Comment:
@Prajwal17Tunerlabs please avoid using new package only for script. we are making our node modules heavy just for one script execution